## DP: Fix compilation error due to flash changes
## DP: Patch author: Alessandro Zummo <a.zummo@towertech.it>
## DP: Upstream status: submitted
# Fixes build failure introduced by
# http://www.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commitdiff;h=54e269ead6e672325866037b0617a72edd1396b9
#

 arch/arm/mach-ixp4xx/nslu2-setup.c |    2 --
 1 file changed, 2 deletions(-)

--- linux-nslu2.orig/arch/arm/mach-ixp4xx/nslu2-setup.c	2006-02-06 21:49:41.000000000 +0100
+++ linux-nslu2/arch/arm/mach-ixp4xx/nslu2-setup.c	2006-02-06 21:59:55.000000000 +0100
@@ -27,8 +27,6 @@ static struct flash_platform_data nslu2_
 };
 
 static struct resource nslu2_flash_resource = {
-	.start			= NSLU2_FLASH_BASE,
-	.end			= NSLU2_FLASH_BASE + NSLU2_FLASH_SIZE,
 	.flags			= IORESOURCE_MEM,
 };
 
